packages/lab/stories/dropdown-next/dropdown-next.stories.tsx

```diff
@@ -0,0 +1,147 @@
+import { Story, ComponentMeta } from "@storybook/react";
+
+import { DropdownNext, DropdownNextProps } from "@salt-ds/lab";
+import { Button } from "@salt-ds/core";
+import { ArrowDownIcon, ArrowUpIcon } from "@salt-ds/icons";
+import { useRef, useState } from "react";
+
+export default {
+  title: "Lab/Dropdown Next",
+  component: DropdownNext,
+} as ComponentMeta<typeof DropdownNext>;
+
+const SimpleListExample = [
+  "Alabama",
+  "Alaska",
+  "Arizona",
+  "Arkansas",
+  "California",
+  "Connecticut",
+  "Delaware",
+  "Florida",
+  "Georgia",
+];
+
+const ListExample = [
+  { value: "🍊", id: "Orange" },
+  { value: "🍏", id: "Apple" },
+  {
+    value: "🍈",
+    id: "Melon",
+  },
+  { value: "🍍", id: "Pineapple" },
+  { value: "🍐", id: "Pear" },
+];
+
+const DropdownTemplate: Story<DropdownNextProps<T>> = ({ ...args }) => {
+  return <DropdownNext {...args} />;
+};
+
+export const Default = DropdownTemplate.bind({});
+Default.args = {
+  source: SimpleListExample,
+};
+
+export const Secondary = DropdownTemplate.bind({});
+Secondary.args = {
+  variant: "secondary",
+  defaultSelected: `${ListExample[2].value} ${ListExample[2].id}`,
+  source: ListExample.map((item) => {
+    return `${item.value} ${item.id}`;
+  }),
+};
+
+export const Readonly = DropdownTemplate.bind({});
+Readonly.args = {
+  readOnly: true,
+  defaultSelected: `${ListExample[3].value} ${ListExample[3].id}`,
+  source: ListExample.map((item) => {
+    return `${item.value} ${item.id}`;
+  }),
+};
+
+export const Disabled = Default.bind({});
+Disabled.args = {
+  disabled: true,
+  defaultSelected: SimpleListExample[7],
+  source: SimpleListExample,
+};
+
+export const Controlled: Story<DropdownNextProps<T>> = ({
```

Comment:
In this example when there isn't enough space at the bottom of the page floating ui renders the drop down above the input and so covers the control buttons. Could be worth mentioning in the docs that users should be aware of if they want to re-create the "controlled" option.